### PR TITLE
[TR] Merge from 1.2

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

**Comment by solivier** [comment]
GTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -395,7 +395,7 @@
 - PIM-3632: Correctly show scopable attribute icons on scope change
 - PIM-3583: Fix the bad parsed filter value with spaces
 
-# 1.2.x
```
Comment (grena) [inline CHANGELOG.md]: This should stay, no?

**Comment by grena** [comment]
Commented, GTM once "resolved"

File: CHANGELOG.md (hunk 1 above)
Comment (phaseinducer) [inline CHANGELOG.md]: I'm not sure, I think it was an error during the previous merge. If you look at the file in 1.2 (https://github.com/akeneo/pim-community-dev/blob/1.2/CHANGELOG.md) you can see that PIM-4227 was fixed in 1.2.35. I think we should see the same thing in 1.3.

Comment (grena) [inline CHANGELOG.md]: Ok !

**Pull request merged**